### DISCUSSION TITLE
TrixiePup64: failed generation of bdrv

### DISCUSSION THREAD

**Discussion by radky2** [puppylinux-woof-CE/woof-CE#4188] [Q&A] [answered]
If I understand correctly, the bdrv builds without errors for VanillaDpup 11.0.

My initial local build of TrixiePup64 11.0 works nicely when BUILD_BDRV=no in _00build.conf, but fails when BUILD_BDRV=yes.

I see the following errors when generating the bdrv (full log attached below):

Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libelogind0 : Conflicts: systemd but 254.5-1 is to be installed
 polkitd : Depends: default-logind orlogind
 procps : Depends: libsystemd0 (>= 254~rc1)
 systemd : Depends: libsystemd0 (= 254.5-1)
E: Unable to correct problems, you have held broken packages.

```
[TrixiePup64_build_log.txt](https://github.com/puppylinux-woof-CE/woof-CE/files/13417124/TrixiePup64_build_log.txt)
```
The following libelogin0 and systemd packages are successfully downloaded to the TrixiePup64 local-build repositories:

/tmp/Trixiepup64_build/local-repositories/x86_64/packages-deb-trixie/libelogind0_252.9-1debian2_amd64.deb
/tmp/Trixiepup64_build/local-repositories/x86_64/packages-deb-trixie/libsystemd-shared_254.5-1_amd64.deb
/tmp/Trixiepup64_build/local-repositories/x86_64/packages-deb-trixie/systemd_254.5-1_amd64.deb
/tmp/Trixiepup64_build/local-repositories/x86_64/packages-deb-trixie/systemd-dev_254.5-1_all.deb

Thanks

**Comment by dimkr** [comment, marked as answer]
See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1052064.

Packages that depends on libsystemd cannot be installed on a system with sysvinit instead of systemd, because libelogind is too old (<254) and some packages (like procps) need (libsystemd|libelogind)>=254.

Once this is fixed in Debian, build should pass.

> If I understand correctly, the bdrv builds without errors for VanillaDpup 11.0.

It's built using a [fork of woof-CE](https://github.com/vanilla-dpup/woof-CE) that allows systemd in bdrv. The build passes but the result is much bigger. This is a temporary solution until elogin in trixie is usable again.

In general, don't be surprised if trixie breaks at least every once in a while, while it's so far from release. sid breaks even more often.

**Reply by radky2** [reply]
Thanks for the quick response !

**Reply by dimkr** [reply]
Backported the workaround from my fork, d6733d6

**Reply by radky2** [reply]
Just finished a new build of TrixiePup64 with your workaround...

To my surprise the bdrv is only 7948K but works fine in initial testing.

Thanks dimkr !

**Reply by dimkr** [reply]
Yes, it should work fine, although it's much bigger.

**Reply by tosim91** [reply]
THANK YOU VERY  MUCH guys for what you are doing! Yea Puppy!
